includes/Core/Authentication/Authentication.php

```diff
@@ -703,9 +703,9 @@ private function get_rest_routes() {
 							$access_token = $oauth_client->get_access_token();
 
 							$data = array(
-								'isAuthenticated' => ! empty( $access_token ),
-								'requiredScopes'  => $oauth_client->get_required_scopes(),
-								'grantedScopes'   => ! empty( $access_token ) ? $oauth_client->get_granted_scopes() : array(),
+								'authenticated'  => ! empty( $access_token ),
```

Comment:
Only here this actually needs to be changed (plus references to here of course).